android/expoview/src/main/java/host/exp/exponent/notifications/exceptions/UnableToScheduleException.java

```diff
@@ -0,0 +1,3 @@
+package host.exp.exponent.notifications.exceptions;
+
+public class UnableToScheduleException extends Exception { }
```

Comment:
Why don't we extend `CodedException` from `unimodules-core`?

Why don't we override a default message for this exception to be more readable and descriptive?